changes/3846.changed

```diff
@@ -0,0 +1 @@
+Changed `VirtualChassis.name` to be globally unique and to serve as the natural key for this model.
```

Comment:
Please also document in `nautobot/docs/installation/tables/v2-database-behavior-changes.yaml`